rust/main/chains/hyperlane-ton/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "hyperlane-ton"
+documentation.workspace = true
+edition.workspace = true
+homepage.workspace = true
+license-file.workspace = true
+publish.workspace = true
+version.workspace = true
+
+[dependencies]
+async-trait.workspace = true
+serde.workspace = true
+serde_json.workspace = true
+thiserror.workspace = true
+anyhow.workspace = true
+log.workspace = true
+derive-new.workspace = true
+hex.workspace = true
+url = { workspace = true }
+pretty_env_logger = {workspace = true}
+base64 = {workspace = true}
+tracing = {workspace = true}
+num-bigint = {workspace = true}
```

Comment:
Could we please use `.workspace = true` format if we don't need to mention features?

Reply:
Yes, sure.